BuildingBlocks/src/BuildingBlocks.API/Serilog/IdentityAddressMaskingOperator.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+public class IdentityAddressMaskingOperator : RegexMaskingOperator
+{
+    public IdentityAddressMaskingOperator(string regexString) : base(regexString) { }
+    private static string IdentityAddressRegex => IdentityAddress.IdentityAddressValidatorRegex().ToString();
+    public static IdentityAddressMaskingOperator Create() => new(IdentityAddressRegex[1..^1]);
```

Comment:
Do you expect anyone who reads this code to understand why you are removing the first and last character?
Also, relying on the first and last characters to be the `^` and `$` is pretty unsecure.

Reply:
Ok, I can leave the `Substring` call.

My idea was that, since `^` and `&` are the start and the end of string, and the validator is checking the individual isolated identity addresses, they will always be present. If that is not acceptable, I can check if characters at indices `0` and `length-1` are really `^` and `&` and only remove them if they are. 

However, it sounds to me like it should be a concern of whoever changes the regex in the future, since there could be other characters in the beginning or the end that are vital, but my mask would still remove them. I guess whoever updates the regex should take care to use the `^` and `&` tokens appropriately.

Reply:
> My idea was that, since ^ and & are the start and the end of string, and the validator is checking the individual isolated identity addresses, they will always be present. If that is not acceptable, I can check if characters at indices 0 and length-1 are really ^ and & and only remove them if they are.
> 
> However, it sounds to me like it should be a concern of whoever changes the regex in the future, since there could be other characters in the beginning or the end that are vital, but my mask would still remove them. I guess whoever updates the regex should take care to use the ^ and & tokens appropriately.

Hmm. Yeah okay, I can go with that.

Reply:
> Do you expect anyone who reads this code to understand why you are removing the first and last character?

This still applies

Reply:
I could've sworn I've already addressed that. It's taken care of now.

Reply:
Ah, yes... We have a problem. I actually used the `Substring` initially, but then, when my formatting test failed, I ran `dotnet format` and that's what changed the expression to `IdentityAddressRegex[1..^1]`. In other words, the `Substring` call makes the formatting test fail.

Reply:
Ah, ok, I understand what you meant. Changes made.